frontend/components/page-contents/feedback-task-content/RecordFeedbackTaskAndQuestionnaire.content.vue

```diff
@@ -50,7 +50,8 @@ import {
   isRecordWithRecordIndexByDatasetIdExists,
   isAnyRecordByDatasetId,
 } from "@/models/feedback-task-model/record/record.queries";
-
+import { deleteRecordResponsesByUserIdAndResponseId } from "@/models/feedback-task-model/record-response/recordResponse.queries";
+import { deleteAllRecordFields } from "@/models/feedback-task-model/record-field/recordField.queries";
```

Comment:
Be sure that these functions are returning a promise. I had problems with `deleteAllRecords` functions with had no `async-await` definition.